### PR TITLE
Add command block cart into creative tab and fixed a cart summon bug

### PR DESCRIPTION
**Pull request by liach**
Sorry CJ. This time I've tested and it really works.

Other: Update gradle wrapper and fixed bug with idea textures (Believe it or not it is really fixed now)

### CODE REVIEW

**Comment by CovertJaguar** [comment]
I'm hesitant to change the build script as it all works fine in IDEA for me.

File: src/main/java/mods/railcraft/common/modules/ModuleCore.java [hunk 1]
```diff
@@ -213,6 +214,7 @@ public void initFirst() {
 
         // Items
         replaceVanillaCart(EnumCart.COMMAND_BLOCK, Items.command_block_minecart, "MinecartCommandBlock", 40);
+        Items.command_block_minecart.setCreativeTab(CreativeTabs.tabTransport);
```
Comment (CovertJaguar) [inline src/main/java/mods/railcraft/common/modules/ModuleCore.java]: This shouldn't be necessary. All minecart items default to the transport tab...

Comment (liach) [inline src/main/java/mods/railcraft/common/modules/ModuleCore.java]: But it does not show up in vanilla.

**Comment by liach** [comment]
That line `idea.modules.inheritOutputDirs = true` works for me, and was suggested by Abrarsyed.

**Comment by CovertJaguar** [comment]
well...I can try but the last time I tried something like this it broke release building.

**Comment by CovertJaguar** [comment]
And when I redeployed my dev env back in August it worked out of the box without any special build targets in the IDE. So I'm not really clear on what problem is.

**Comment by liach** [comment]
So should I test this with eclipse?

**Comment by CovertJaguar** [comment]
Well you can if you want, but I don't really care about Eclipse. I'm using IntelliJ same as you.

**Comment by liach** [comment]
Now does it hurt? Do I need to remove that line?

**Pull request merged**